### PR TITLE
Steps 1-4 Public Feed feature

### PR DESCRIPTION
**Pull request by lmtrujillo**

### CODE REVIEW

File: .factorypath [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<factorypath>
```
Comment (justincuig) [inline .factorypath:1]: what's the changes in this path used for?

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+
+        response.setContentType("text/html");
+
+        String user = request.getParameter("user");
+
+        if(user == null || user.equals("")) {
```
Comment (justincuig) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: these keywords should be followed by an empty space for better readability

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        // Display the aboutMe string on the page at http://localhost:8080/about?user=<user>
+        response.getOutputStream().println(aboutMe);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+
+        UserService userService = UserServiceFactory.getUserService();
+        if (!userService.isUserLoggedIn()) {
```
Comment (justincuig) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: this is good

**Review by justincuig — CHANGES_REQUESTED** [review]

File: .factorypath (hunk 1 above)
Comment (lmtrujillo) [reply .factorypath:1]: I don't remember making modifications to that file

Comment (lmtrujillo) [reply .factorypath:1]: Should I erase it or make modifications?

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java (hunk 2 above)
Comment (lmtrujillo) [reply src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: I don't understand what you are asking, could you be a bit more specific please?

File: src/main/java/com/google/codeu/servlets/MessageFeedServlet.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+  response.setContentType("application/json");
+  
+  List<Message> messages = datastore.getAllMessages();
+  Gson gson = new Gson();
+  String json = gson.toJson(messages);
+  
+  response.getOutputStream().println(json);
+ }
+}
```
Comment (mpwint) [inline src/main/java/com/google/codeu/servlets/MessageFeedServlet.java:43]: This looks good

**Review by mpwint — APPROVED** [review]
I am assuming that <factorypath> is generated from your IDE. Other than that, things look good.

**Review by justincuig — APPROVED** [review]

**Pull request merged**